src/support/ios.c

```diff
@@ -1078,7 +1078,7 @@ int ios_putc(int c, ios_t *s)
 
 int ios_getc(ios_t *s)
 {
-    char ch;
+    char ch = 0;
```

Comment:
I don't think this should trigger valgrind. As far as I can tell, it's initialized on all paths that read it.

Reply:
Perhaps valgrind is being too cautious? Making this change did squash the valgrind error.

Reply:
> As far as I can tell, it's initialized on all paths that read it.

That may be why Clang doesn't complain then, but it definitely has the `-Wsometimes-uninitialized` warning (but GCC [doesn't](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=18501)).